Code/Legacy/CryCommon/WinBase.cpp

```diff
@@ -1264,7 +1264,8 @@ const bool GetFilenameNoCase
 (
     const char* file,
     char* pAdjustedFilename,
-    const bool cCreateNew
+    const bool /*cCreateNew
+*/
```

Comment:
Based on the usage of `GetFilenameNoCase` it looks like we can remove cCreateNew parameter entirely from the declaration and definition.